api/client/search.go

```diff
@@ -76,7 +76,7 @@ func (cli *DockerCli) CmdSearch(args ...string) error {
 
 		}
 		fmt.Fprint(w, "\t")
-		if res.IsAutomated || res.IsTrusted {
+		if res.IsAutomated || res.IsOfficial {
```

Comment:
I don't think this should be `IsOfficial`. `IsTrusted` was the same as `IsAutomated` before.